docker-compose.yml

```diff
@@ -1,37 +1,51 @@
 ---
-version: "3.7"
-
-# This docker-compose file is used to build and test the container
-
-secrets:
-  quote_txt:
-    file: ./src/secrets/quote.txt
-
+version: '3'
 services:
-  example:
-    # Run the container normally
+
+  li-pca-api:
     build:
-      # VERSION must be specified on the command line:
-      # e.g., --build-arg VERSION=0.0.1
       context: .
-      dockerfile: Dockerfile
-    image: cisagov/example
```

Comment:
You should update this instead of removing it entirely.

Reply:
Addressed in: c617532